roles/common/tasks/user_config.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Set default shell for the user
```

Comment:
To keep it cleaner add it to the oh-my-zsh config where you configure your shell

Reply:
https://gitlab.com/cryptolukas/ansible-local/-/blob/master/local/tasks/zsh.yml

Reply:
I added this but even with it ansible tries to use /bin/sh